klippa/src/cpal.rs

```diff
@@ -0,0 +1,27 @@
+//! impl subset() for CPAL table
+
+use crate::{NameidClosure, Plan};
+use write_fonts::read::{tables::cpal::Cpal, types::NameId};
+
+impl<'a> NameidClosure for Cpal<'a> {
```

Comment:
Small nit: can we rename this trait to `NameIdClosure`?

I'm parsing "Nameid" as some sort of Greek hero which is spoiling our Old Norse theme for me :)

Reply:
Done.